rest/src/main/kotlin/service/InteractionService.kt

```diff
@@ -16,11 +18,19 @@ class InteractionService(requestHandler: RequestHandler) : RestService(requestHa
     suspend fun createGlobalApplicationCommand(
         applicationId: Snowflake,
         request: ApplicationCommandCreateRequest
-    ): DiscordApplicationCommand = call(Route.GlobalApplicationCommandCreate) {
+    ) = call(Route.GlobalApplicationCommandCreate) {
```

Comment:
```suggestion
    ): DiscordApplicationCommand = call(Route.GlobalApplicationCommandCreate) {
```

Unnecessary removal of return type, we're gonna have to specify them all explicitly anyway once we switch to library mode.

Reply:
Then shouldn't all types be explictly set in all functions in the `Interactions.kt` class? Most of them do not have any explict types so that's why I removed them from those two functions.

I can readd them if needed.

Reply:
> Then shouldn't all types be explictly set in all functions in the InteractionServices.kt class? 

Eventually, they should be set everywhere, I'm not going to force people to do the extra work though. It's not a big deal either way.